napari/layers/points/points.py

```diff
@@ -1009,9 +1009,12 @@ def selected_data(self, selected_data):
             with self.block_update_properties():
                 self.current_size = size
 
-        properties = {
-            k: np.unique(v[index], axis=0) for k, v in self.properties.items()
-        }
+        properties = {}
+        for k, v in self.properties.items():
+            # only use axis argument if strictly necessary
```

Comment:
Optional: I think the line of code below this speaks for itself, so I would just remove this implementation comment. Or I would consider explaining *why* using `axis=None` is preferred by default (i.e. to support `dtype=object`), so that someone doesn't change this in the future (though we have the test to catch that too).

Reply:
Good point!

Reply:
Changed to a brief explanation of the pandas issue.